process_pr.py

```diff
@@ -1043,6 +1043,9 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
                     )
                     issue.create_comment(msg)
             return
+        if pr.merged and (not force):
+            logger.error("This PR is merged, quitting")
+            return
```

Comment:
adding this check here means bot will not process any comment ... right? what is we want to assign a label for a merged/closed PR?

Reply:
> adding this check here means bot will not process any comment ... right? 

yes

> what if we want to assign a label for a merged/closed PR?

We can run the bot with force flag (I don't think this will happen often). But I can try blocking less if we want to support this usage scenario.

Reply:
we do want to process all comments even if PR is merged/closed. Many L2 signs a PR after it has been merged. so force running is not an option.

I would suggest to disable setting commit status and running PR tests if PR is merged/closed and old commit statuses are empty